src/Authenticator/JwtAuthenticator.php

```diff
@@ -89,7 +89,7 @@ public function authenticate(ServerRequestInterface $request): ResultInterface
 
         $result = json_decode(json_encode($result), true);
 
-        $key = IdentifierInterface::CREDENTIAL_JWT_SUBJECT;
+        $key = $this->getConfig('dataField', IdentifierInterface::CREDENTIAL_JWT_SUBJECT);
```

Comment:
You should add the new key in the default config array using the constant for it's default value. Also `dataKey` might be a better name.

Reply:
done